mthamayor/auto-mart
# #166715879 #166715846 Created auth database relations

#### What does this PR do?
Implement database requirement to some Auth endpoints

#### Description of Task to be completed?
Switch from using dummyDB to a persistent Postgres database for the following endpoints
1. **POST /auth/signup**
2. **POST /auth/signin**

#### How should this be manually tested?
- Clone the repo and CD into it
- Install dependencies with `npm install`
- On your machine, create a database named **'auto_mart'** and make sure the user **'postgres'** has full privileges
- Drop all existing tables in the database by running the following command on your Command Line Interface 
```
npm run db:reset
```
- Create tables the database by running the following command on your Command Line Interface 
```
npm run db:migrate
```
- You can now test the above-specified endpoints on postman
Check [the documentation](https://mthamayor-auto-mart.herokuapp.com/api/v1/docs) for the endpoints specifications

#### Any background context you want to provide?
Postgres database should have been installed on the machine being used

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166715846
https://www.pivotaltracker.com/story/show/166715879

#### Screenshots (if appropriate)
![Screenshot (124)](https://user-images.githubusercontent.com/16121915/59566424-12e48a00-9058-11e9-9fb1-b659b94d6607.png)

> db:reset and db:migrate

#### Questions:
None
